ci/vmtest/run_selftests.sh

```diff
@@ -54,21 +48,34 @@ read_test_names() {
     foldable end read_test_names
 }
 
-test_progs() {
-  foldable start test_progs "Testing test_progs"
+test_progs_helper() {
+  local suffix="$1"
+  local args="$2"
+  local selftest="test_progs${suffix}"
+
+  foldable start test_progs${suffix} "Testing test_progs${suffix}"
```

Comment:
minor: can we reuse $selftest variable here?

Reply:
Indeed, we should.